components/micro-gateway-cli/src/main/resources/default-deployment-config.toml

```diff
@@ -98,18 +98,19 @@
     #cpuPrecentage = ''
   [kubernetes.kubernetesSecret]
     enable = false
-    #name = ''
-    #mountPath = ''
-    #readOnly = ''
-    #data = ''
+    #[[kubernetes.kubernetesSecret.secrets]]
+      #name = ''
+      #mountPath = ''
+      #readOnly = false
+      #data = ['']
   [kubernetes.kubernetesConfigMap]
     enable = false
     ballerinaConf = 'path/to/micro-gw.conf'
     #[[kubernetes.kubernetesConfigMap.configMaps]]
-    #    name = ''
-    #    mountPath = ''
-    #    readOnly = false
-    #    data = ['']
+      #name = ''
+      #mountPath = ''
+      #data = ['']
+      #readOnly = false
   [kubernetes.kubernetesPersistentVolumeClaim]
```

Comment:
Lets add a label example to the default config

Reply:
Fixed with 43012f6